modules/osdk-golang-run-in-cluster.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-run-in-cluster_{context}"]
+= Running as a deployment inside the cluster
+
+You can run your Operator project as a deployment inside of your cluster.
+
+.Procedure
+
+. *Build and push the image.*
+
+.. Before building the Operator image, ensure the generated Dockerfile references the base image you want. You can change the default "runner" image `gcr.io/distroless/static:nonroot` by replacing its tag with another, for example `alpine:latest`, and removing the `USER: nonroot:nonroot` directive.
```

Comment:
```suggestion
.. Before building the Operator image, ensure that the generated Dockerfile references the base image that you want. You can change the default "runner" image, `gcr.io/distroless/static:nonroot`, by replacing its tag with another, for example `alpine:latest`, and removing the `USER: nonroot:nonroot` directive.
```

Reply:
Broke this apart per other Eng feedback.